feature-libs/user/profile/components/register/register.component.html

```diff
@@ -211,6 +211,10 @@
               </label>
             </div>
           </div>
+          <cx-captcha (confirmed)="captchaConfirmed()"></cx-captcha>
```

Comment:
The function `captchaConfirmed()` is used to set `registerForm.captcha`. It will help to `handle cx-form-errors`.